static/src/js/components/SearchPanels/SearchPanels.js

```diff
@@ -726,6 +726,8 @@ class SearchPanels extends PureComponent {
 
     const { edscHost } = this
 
+    // TODO: does this need toe lbe lazy loaded
```

Comment:
typo?

Reply:
I just wanted to talk it through during the PR